cli/src/lib.rs

```diff
@@ -0,0 +1,283 @@
+use std::io::{Result, Error, ErrorKind};
+use std::sync::Arc;
+use std::sync::mpsc::{channel, Sender, Receiver};
+
+use log::{info, error, LevelFilter};
+use log4rs::append::rolling_file::RollingFileAppender;
+use log4rs::encode::pattern::PatternEncoder;
+use log4rs::config::{Appender, Config, Root};
+use log4rs::filter::threshold::ThresholdFilter;
+use log4rs::append::rolling_file::policy::compound::{
+    CompoundPolicy,
+    trigger::size::SizeTrigger,
+    roll::fixed_window::FixedWindowRoller,
+};
+
+
+use zecwalletlitelib::{commands,
+    lightclient::{LightClient, LightClientConfig},
+};
+
+#[macro_export]
+macro_rules! configure_clapapp {
+    ( $freshapp: expr ) => {
+    $freshapp.version("1.0.0")
+            .arg(Arg::with_name("dangerous")
+                .long("dangerous")
+                .help("Disable server TLS certificate verification. Use this if you're running a local lightwalletd with a self-signed certificate. WARNING: This is dangerous, don't use it with a server that is not your own.")
+                .takes_value(false))
+            .arg(Arg::with_name("nosync")
+                .help("By default, zecwallet-cli will sync the wallet at startup. Pass --nosync to prevent the automatic sync at startup.")
+                .long("nosync")
+                .short("n")
+                .takes_value(false))
+            .arg(Arg::with_name("recover")
+                .long("recover")
+                .help("Attempt to recover the seed from the wallet")
+                .takes_value(false))
+            .arg(Arg::with_name("plantseed")
```

Comment:
Why is this named "plant seed"? It should just be seed

Reply:
I thought it was confusing that there was both an OPTION and a COMMAND with the same name.

`zecwallet-cli --seed <seed_phrase>` creates a new wallet

`zecwallet-cli seed`  displays the wallet's seed

I can revert the change if you like, I thought it was worth discussing the name collision.